panel/widgets/select.py

```diff
@@ -36,6 +36,8 @@
 from .input import TextAreaInput, TextInput
 
 if TYPE_CHECKING:
+    import pandas as pd
```

Comment:
Should I just import pandas outside of TYPE_CHECKING?

Reply:
No I think it should be just in TYPE_CHECKING.